openquake/engine/utils/tasks.py

```diff
@@ -67,6 +67,7 @@ def aggregate_result_set(self, agg, acc):
         if not self.results:
             return acc
         backend = current_app().backend
+        amqp_backend = backend.__class__.__name__.startswith('AMQP')
```

Comment:
I prefer to do this, since the AMQP class has changed location between celery 2 and 3, so we would need a conditional import. This is a temporary hack anyway, we can improve on that when we drop the support for celery 2.